webrender/src/frame.rs

```diff
@@ -117,6 +117,7 @@ impl<'a> FlattenContext<'a> {
         self.flatten_items(traversal, pipeline_id, LayerVector2D::zero());
 
         if self.builder.config.enable_scrollbars {
```

Comment:
Maybe the option can be renamed to more accurately reflect what it does if my change is accepted?